Add video tutorials to Bascis/Accounts (#4038)
w3f/polkadot-wiki: docs/learn/learn-account-generation.md

understand the risks of the account generation method you choose and how to properly mitigate them
in order to keep your funds safe.<br/><br/>

> :::info Visit [this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely) for more information about key security.

Reviewer: ```suggestion
:::info 

Visit [this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely) for more information about key security.
```

It is still running on the same computer you use to connect to the internet with and thus is less
secure than using Parity Signer or other air-gapped approaches.

### Reset Password using the Browser Extension

:::info

> See [**this video tutorial**](https://www.youtube.com/watch?v=DNU0p5G0Gqc) to learn how to change the password for an account that has been created on the Polkadot-JS browser extension (i.e. an injected account).

Reviewer 2: We can also specify the timestamp for the videos that take the viewer directly to the specific chapter in the video, especially if the topic appears later in the video.

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/86818441/201632286-2b2298cd-b158-459a-b3e4-4af67e17e5f7.png">

```suggestion
See [**this video tutorial**](https://www.youtube.com/watch?v=DNU0p5G0Gqc&t=280s) to learn how to change the password for an account that has been created on the Polkadot-JS browser extension (i.e. an injected account).
```

:::info

> See [**this video tutorial**](https://www.youtube.com/watch?v=DNU0p5G0Gqc) to learn how to change the password for an account that has been created on the Polkadot-JS UI (i.e. a non-injected account).

Reviewer 2: ```suggestion
See [**this video tutorial**](https://youtu.be/DNU0p5G0Gqc?t=261) to learn how to change the password for an account that has been created on the Polkadot-JS UI (i.e. a non-injected account).
```

## Parity Signer

:::info

> For guidelines about how to create an account using Parity Signer, see [**this video tutorial**](https://youtu.be/hgv1R9mPEXw) and visit [**this support article**](https://support.polkadot.network/support/solutions/articles/65000180512-how-to-create-an-account-in-parity-signer).

Reviewer 2: ```suggestion
For guidelines about how to create an account using Parity Signer, see [**this video tutorial**](https://youtu.be/hgv1R9mPEXw?t=120) and visit [**this support article**](https://support.polkadot.network/support/solutions/articles/65000180512-how-to-create-an-account-in-parity-signer).
```
